server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraCacheConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package org.apache.james.blob.cassandra.cache;
+
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+import javax.inject.Inject;
+
+import org.apache.commons.configuration2.Configuration;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public class CassandraCacheConfiguration {
+
+    private final String TIME_OUT_PROPERTY = "";
+    private final String MAX_SIZE_PROPERTY = "";
+
+    private final long DEFAULT_TIME_OUT = 50;
+    private final int DEFAULT_MAX_SIZE = 8;
+
+    final Duration TIME_OUT;
+    final int MAX_SIZE;
+
+    @Inject
+    public CassandraCacheConfiguration(Configuration propertiesConfiguration) {
+        this.TIME_OUT = Duration.of(propertiesConfiguration.getLong(TIME_OUT_PROPERTY, DEFAULT_TIME_OUT), ChronoUnit.MILLIS);
```

Comment:
DEFAULT_TIME_OUT should be a Duration. Also use DurationParser or you'll hear complain later :)

Reply:
Let's keep Duration parsing & guice injection topics for later ;-) (it's next ticket)